src/ResourceManager/Sql/Commands.Sql/help/Clear-AzSqlInstanceDatabaseVulnerabilityAssessmentRuleBaseline.md

```diff
@@ -0,0 +1,211 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version: https://docs.microsoft.com/en-us/powershell/module/az.sql/clear-azsqlinstancedatabasevulnerabilityassessmentrulebaseline
+schema: 2.0.0
+---
+
+# Clear-AzSqlInstanceDatabaseVulnerabilityAssessmentRuleBaseline
+
+## SYNOPSIS
+Clears the vulnerability assessment rule baseline.
+
+## SYNTAX
+
+### GetByNameParameterSet
+```
+Clear-AzSqlInstanceDatabaseVulnerabilityAssessmentRuleBaseline [-InstanceName] <String>
+ [-DatabaseName] <String> -RuleId <String> [-RuleAppliesToMaster] [-ResourceGroupName] <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ClearByParentResourceParameterSet
+```
+Clear-AzSqlInstanceDatabaseVulnerabilityAssessmentRuleBaseline -InputObject <AzureSqlManagedDatabaseModel>
+ -RuleId <String> [-RuleAppliesToMaster] [-ResourceGroupName] <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Clear-AzSqlInstanceDatabaseVulnerabilityAssessmentRuleBaseline** cmdlet removes the vulnerability assessment rule baseline.
+Note that you need to run *Enable-AzSqlInstanceAdvancedThreatProtection* and *Update-AzSqlInstanceDatabaseVulnerabilityAssessmentSettings* cmdlet as a prerequisite for using this cmdlets.
+
+## EXAMPLES
+
+### Example 1 - Clears the vulnerability assessment rule baseline
+```powershell
+PS C:\> Clear-AzSqlInstanceDatabaseVulnerabilityAssessmentRuleBaseline `
+            -ResourceGroupName "ResourceGroup01" `
+            -InstanceName "ManagedInstance01" `
+            -DatabaseName "Database01" `
+            -RuleId "VA2108"
+```
+
+### Example 2 - Clears the vulnerability assessment rule baseline from a managed database object
+```powershell
+PS C:\> Get-AzSqlInstanceDatabase `
+            -ResourceGroupName "ResourceGroup01" `
+            -InstanceName "ManagedInstance01" `
+            -Name "Database01" `
+            | Clear-AzSqlInstanceDatabaseVulnerabilityAssessmentRuleBaseline `
+                -RuleId "VA2108" `
+                -AppliesToMaster
+```
+
+## PARAMETERS
+
+### -DatabaseName
+SQL Managed Database name.
+
+```yaml
+Type: String
```

Comment:
You need to regenerate your docs with the `-UseFullTypeName` flag. See the documentation here: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module

Reply:
I regenerated them again